packages/web/src/scss/components/Grid/_tools.scss

```diff
@@ -47,3 +48,52 @@
         }
     }
 }
+
+@function suffix-breakpoint-class($name, $value) {
+    @if $value > 0 {
+        @return unquote('-' + $name);
+    }
+
+    @return '';
+}
+
+// Generates grid-column values for each breakpoint with fallbacks to the previous breakpoint.
+// Either end or span must be specified, but if both are specified, end will be used.
+@mixin item($breakpoints) {
+    $column-start: auto;
+    $column-span: auto;
+    $column-end: '';
```

Comment:
How about `auto`? Isn't it a better default value?

https://developer.mozilla.org/en-US/docs/Web/CSS/grid-column-end

Then we could get rid of the condition on line 77, I believe.

Reply:
nope, it has to be empty, because we place there the span values